fuzzer.scm

```diff
@@ -1,3 +1,7 @@
+(load "staged-mk.scm")
```

Comment:
can you do
```
;; using namin/faster-miniKaren branch staged
(load "../faster-miniKanren/mk-vicare.scm")
(load "../faster-miniKanren/mk.scm")
```

instead of staged-mk loading? It will be faster for all the fuzzers.